client/blocks/inline-help/inline-help-rich-result.jsx

```diff
@@ -72,7 +67,7 @@ class InlineHelpRichResult extends Component {
 				dialogType: 'video',
 				videoLink: link,
 			} );
-		} else if ( type === RESULT_ARTICLE && postId && isLocaleEnglish ) {
+		} else if ( ( ! type || type === RESULT_ARTICLE ) && postId && isLocaleEnglish ) {
```

Comment:
Alternatively, you could default `type = RESULT_ARTICLE` when destructuring from `result`.